explorer/explorerroutes.go

```diff
@@ -1586,6 +1420,8 @@ func (exp *explorerUI) Search(w http.ResponseWriter, r *http.Request) {
 
 	// Try aux DB if in full mode.
 	if !exp.liteMode {
+		// This is be unnecessarily duplicative and possible very
+		// slow for a very active addresss.
```

Comment:
I'd like to personally follow up on this PR because the call to `exp.blockData.GetExplorerAddress` several lines up is no longer appropriate above.  It was simply used to validate the address, but since adding `txhelpers.AddressValidation`, that's what should be done above (avoiding the expensive SearchRawTransactoinVerbose RPC).

The purpose of this "Try aux DB if in full mode" when the RPC was *just used* above was because transactions for orphaned blocks may not show up in the output of searchrawtransactions, but PostgreSQL may have recorded data for the transaction and associated addresses.  This is the same reasoning for checking PostgreSQL for transactions with `exp.explorerSource.Transaction` on [this line below](https://github.com/decred/dcrdata/pull/845/files#diff-022a8370d807e3c88802b53baa33af77R1463).

However, Search does not need to do the address queries at all.  That is the job of the address page.  Here we only need to do `txhelpers.AddressValidation` (as done in `GetExplorerAddress` and elsewhere) and redirect to the appropriate /address URL.

Anyway, I can pick this up after your PR since it's a Search page issue.